tsconfig.json

```diff
@@ -15,8 +18,22 @@
     "suppressImplicitAnyIndexErrors": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "target": "es5",
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true
```

Comment:
Curious, did u mean to add these?

Reply:
I didn't intentionally add them but reviewing the changes that apparently my IDE automatically added they seemed good.

Reply:
i don't think we want `skipLibCheck: true` cuz that means we won't type check our type files :/

Reply:
Ah, ok - I'll remove that then

Reply:
the only thing i'm iffy on other than that is `noEmit`... not exactly sure what it does-- don't wanna hold this PR up too much tho

Reply:
cuz apparently it means that no files are outputted? so its only used to typecheck, not compile, typescript... wondering if our build will still work correctly in that case...

Reply:
https://stackoverflow.com/questions/55002137/typescript-noemit-use-case